docs/StardustDocs/topics/operations/utils/shuffle.md

```diff
@@ -0,0 +1,47 @@
+# shuffle
+
+
+<web-summary>
+Discover `shuffle` operation in Kotlin Dataframe.
+</web-summary>
+
+<card-summary>
+Discover `shuffle` operation in Kotlin Dataframe.
+</card-summary>
+
+<link-summary>
+Discover `shuffle` operation in Kotlin Dataframe.
+</link-summary>
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.utils.ShuffleSamples-->
+
+Returns a new [`DataFrame`](DataFrame.md) with rows in random order.
+
+You can supply a [kotlin.random.Random](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.random/-random/)
+instance for reproducible results.
```

Comment:
probably important to mention that you need to give it the same seed for this, like you show below